examples/2d/elixir_advection_callbacks.jl

```diff
@@ -0,0 +1,154 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+
+# define new structs inside a module to allow re-evaluating the file
+module TrixiExtensionExample
+
+using Trixi
+using DiffEqCallbacks: DiscreteCallback, u_modified!
+
+
+struct ExampleLimiter
```

Comment:
Maybe a high-level overview?
```suggestion
# This is an example implementation for a simple limiter (i.e., a callable that is executed after each Runge-Kutta stage), which records some values each time it is called. Its sole purpose here is to showcase how to implement a limiter-like functionality for Trixi.
struct ExampleLimiter
```

Reply:
Added in 8f2fe48da94f56682ccfd55b2a8bdec0e6052337.